pagespeed/automatic/system_tests/responsive_images.sh

```diff
@@ -35,4 +35,7 @@ start_test rewrite_images can rewrite srcset itself
 URL=$TEST_ROOT/image_rewriting/srcset.html?PageSpeedFilters=+rewrite_images,+debug
 fetch_until -save $URL 'grep -c xPuzzle.*1x.*xCuppa.*2x' 1
 
+start_test rewrite_images_datasrcset can rewrite data-srcset itself
+URL=$TEST_ROOT/image_rewriting/data-srcset.html?PageSpeedFilters=+rewrite_images,+debug
+fetch_until -save $URL 'grep -c srcset.*xPuzzle.*1x.*xCuppa.*2x' 2
```

Comment:
Should we grep for `data-srcset` instead of just `src-set`?

Reply:
Well.... In the html file we have data-srcset and srcset, this second in the amp-img element.
Whit this grep, using srcset we cover both data-srcset and amp-image srcset. Is for that that we need to count 2 in the grep condition. If use data-srcset in the grep we need other fecht_until to test amp-img srcset.... That can be ok but...We need to check the same file 2 times?

Reply:
I see, sounds good to me as-is then, thanks.